content/getting-started/windows.md

```diff
@@ -5,8 +5,14 @@ draft: false
 weight: 3
 ---
 
-This page has information on how to install and use TinyGo on Windows. The only currently supported option is to use the Docker image.
+This page has information on how to install and use TinyGo on Windows 10.
+
+## Quick Install
+
+You must have Go v1.11+ already installed on your machine in order to install TinyGo.
```

Comment:
It's not clear to me from this description that Go has to be installed _inside_ WSL. But as the Linux installation guide already says you have to install Go, I think it can be removed here.

Reply:
Done.